README.md

```diff
@@ -121,13 +121,14 @@ Injects:
   + **Volume Mounts** at `/conjur/secrets` and `/conjur/podinfo`
 
 Requires:
-  + a conjur-connect configMap in the namespace that the sidecar resides in.
-  + The sidecar deployment manifest must include the conjur-connect configmap
+  + a configMap in the namespace that the sidecar resides in. The config map can be the Golden config or the Conjur connect config map.
```

Comment:
Our docs flow has users deploy the Cluster Prep Helm chart to a dedicated Conjur namespace, and clients/sidecars to different namespaces - will the Golden ConfigMap ever be "in the namespace that the sidecar resides in"? Do we plan on updating our guidance?

Reply:
So if sidecar injector is in the dedicated Conjur namespace , it can read the Golden configmap but if it is in it's own namespace it can also read the altername Conjur connect config map. So either was will work.